TSPL.docc/ReferenceManual/Declarations.md

```diff
@@ -980,6 +1040,150 @@ see <doc:Functions#In-Out-Parameters>.
   ```
 -->
 
+#### Borrowing and Consuming Parameters
+
+By default, Swift uses a set of rules
+to automatically manage manage object lifetime across function calls,
```

Comment:
managed used twice here, possibly unintentionally

Reply:
Thanks — removed in 728bb4ea.